articles/libraries/machine_learning/concepts/Qccc.Introduction.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
With where this file is currently located, the documentation for the machine learning library would be published at the URL `https://docs.microsoft.com/quantum/libraries/machine_learning/concepts/qccc.introduction`. By contrast, the chemistry library is documented at `https://docs.microsoft.com/quantum/libraries/chemistry/`.

Thus, I'd suggest moving this document up a folder and renaming `machine_learning` to `machine-learning` (URLs tend to be typeset with `kebab-case` rather than `snake_case`).

Reply:
sounds good to me. Let's avoid the underscores here in the URLs (as we don't do it elsewhere). Also, let's create an index.yaml and a toc.yaml and shorten the URL to something like https://docs.microsoft.com/quantum/libraries/machine-learning. Alex, could you please do that? @alexeib2